Library/Homebrew/dev-cmd/pr-automerge.rb

```diff
@@ -49,7 +49,7 @@ def pr_automerge
       return
     end
 
-    ohai "#{prs.size} matching pull requests:"
+    ohai "#{prs.size} matching pull #{"request".pluralize(prs.size)}:"
```

Comment:
```suggestion
    ohai "#{prs.count} matching pull #{"request".pluralize(prs.count)}:"
```

Reply:
I don't mind changing, but why is `count` preferred to `size`?

Reply:
It reads nicer. 😉

Reply:
> It reads nicer. 😉 

It sounds like a personal preference that we shouldn't be casting on other maintainers.

Reply:
I'd rather not change existing code if there isn't a good reason, so I'll leave it as is.

Reply:
It's not really a personal preference: You can `count` multiple PRs, but you can't measure their `size`.

Reply:
I think it's fine to suggest personal preference. I agree this is easier to read. It'd fine to me either way so I'm approving but I'd rather see "comment and approve" than maintainers such as @reitermarkus being encouraged to not leave these sort of (useful) comments.

Reply:
> I think it's fine to suggest personal preference.

It is sure fine to make suggestions based on personal preferences, but they should be explained (initial comment had no accompanying explanation) and provide at least some reasoning. I agree with the explanation provided later that one can count PRs but can't measure their size but I disagree with "it reads nicer".